src/components/ApifySearch.tsx

```diff
@@ -29,6 +42,89 @@ const collapseResults = (() => {
 const NavigateContext = createContext((...props: any[]) => { throw new Error('The navigate function has not been initialized yet.') });
 export const useNavigate = () => useContext(NavigateContext);
 
+function getResults({ query, props, section }: { query: string, props: any, section?: string }) {
+  return getAlgoliaResults({
```

Comment:
Is this just the old version + section filter, or something else changed? It's hard to tell from the diff...

Reply:
You're right, I prematurely refactored, but it's not required for these changes in the end. will revert + clean up a bit